packages/cli/src/index.ts

```diff
@@ -102,7 +139,8 @@ async function execCmd(args: string[]) {
   let { hasErrored, requiresInstall } = runChecks(
     packagesByName,
     root,
-    shouldFix
+    shouldFix,
+    cli.flags
```

Comment:
By doing it this way, I figure it can be enhanced with something like `cosmiconfig` in the future to read these kinds of options from a file.